qa/tasks/cephfs/test_misc.py

```diff
@@ -65,7 +65,7 @@ def test_fs_new(self):
 
         get_pool_df = self.fs.get_pool_df
         self.wait_until_true(
-                lambda: get_pool_df(self.fs.metadata_pool_name)['objects'] > 0,
+                lambda: get_pool_df(self.fs.metadata_pool_name, timeout=30)['objects'] > 0,
```

Comment:
I need to read more carefully... 

Still, I think it would be neater to catch the RuntimeError (or maybe get_pool_df return none when pool doesn't exist, and handle that) inside this wait_until_true, rather than nesting the waits